flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlReplaceTableAs.java

```diff
@@ -73,36 +69,14 @@
  * AS SELECT * FROM base_table;
  * }</pre>
  */
-public class SqlReplaceTableAs extends SqlCreate implements ExtendedSqlNode {
+public class SqlReplaceTableAs extends SqlCreateTable implements ExtendedSqlNode {
```

Comment:
don't know why it wasn't like that, however based on list of fields and functionality it seems to be  logical